app/views/cases/sar_internal_review/_new_form_common.html.slim

```diff
@@ -0,0 +1,65 @@
+section
+  header
+
+  br
+  br
+    = f.text_field :subject_type, class: 'visually-hidden', 
+      label_options: { class: 'visually-hidden' }
+
+    = f.text_field :flag_for_disclosure_specialists, 
+      class: 'visually-hidden', 
+      label_options: { class: 'visually-hidden' }, 
+      value: 'true'
+
+    = f.radio_button_fieldset :sar_ir_subtype, 
+      choices: ['Timeliness', 'Compliance'], 
```

Comment:
Can this go in the lang file? And other strings below

Reply:
Do you mean the values of the array?

Reply:
Yes and any others on the page that are easy to put in en

Reply:
I'll add these into the en.yml 👍